zkevm-circuits/src/evm_circuit/witness.rs

```diff
@@ -459,6 +493,26 @@ impl Rw {
                 F::zero(),
                 F::zero(),
             ],
+            Self::TxAccessListAccountStorage {
+                rw_counter,
+                is_write,
+                tx_id,
+                address,
+                key,
+                value,
+                value_prev,
+            } => [
+                F::from(*rw_counter as u64),
+                F::from(*is_write as u64),
+                F::from(RwTableTag::TxAccessListAccountStorage as u64),
+                F::from(*tx_id as u64),
+                address.to_scalar().unwrap(),
+                RandomLinearCombination::random_linear_combine(key.to_le_bytes(), randomness),
+                F::from(*value as u64),
```

Comment:
do need to random_linear_combine of `value` `value_prev` or already done when passing as parameter ?

Reply:
key value value_prev should all be RLCed.     codes here <https://github.com/scroll-tech/zkevm-circuits/pull/100/files#diff-09bf944a4f9216c1053a271fea0ff2f7a985380789b990f8e4ac530159214df2R597> can be copied

Reply:
in TxAccessList value & value_prev are `boolean` instead of `word`.

as in  `TxAccessListAccount ` we don't do random_linear_combine, so I don't think we should do random_linear_combine in `TxAccessListAccountStorage ` here